integration/experiment/power_sweep/gen_policy_recommendation.py

```diff
@@ -176,7 +176,7 @@ def evaluate(self, PL, full=False):
         dev = (sum([val**2 for val in vals])/len(vals) - mean**2)**0.5
         res = (sum([mm.residual() for mm in self._model])/len(self._model))
         if full:
-            return mean, [dev, res]
+            return {'mean': mean, 'dev': [dev, res]}
```

Comment:
why not:
```
return {'mean': mean, 'sigma': rootssq([dev, res])}
```
it seems like sigma is used throughout the rest of the code?  Perhaps there will be another use for each of the terms in the future, or this is for compatibility with another python interface?

Reply:
Nevermind, I see where the residual is used independently below.